src/patternfly/demos/Page/page-template-header-tools-notification-badge.hbs

```diff
@@ -2,18 +2,18 @@
   {{#if page-header-tools--IsNotificationSelected}}
```

Comment:
Is this file being used any more ? you could remove it, and add it to the notification drawer specifically

Reply:
It's included in `page-template-header-tools-elements.hbs` and conditionally rendered, I'd like to keep in case we have other demos that might use it